scipy/constants/__init__.py

```diff
@@ -7,6 +7,9 @@
 
 Physical and mathematical constants and units.
 
+.. raw:: html
```

Comment:
Here `html` indicates the Sphinx "writer". So this will work (once the file is found) for the html docs, but not the pdf docs. @mkg33 I think you tried working with pdf output right? Can you remind me of the details?

Reply:
Thanks so much for trying to make sense of this mess. The thing about the PDF solution was this, quote from my e-mail:

> So here's another solution: including the trees as pure PDFs. I've browsed the Net for similar solutions and it doesn't seem unreasonable (indeed, people have done this and apparently succeeded). Would this idea be acceptable? I have to admit, it would be great quality-wise and would facilitate maintenance. Another way would be exporting the tikzpictures as SVG - preserves the quality (though, again, not the font, but I can fix that) and is browser-friendly (in the mainstream sense).

And we later discussed the output format and you said that the SVG solution was better and that's what we'd agreed on initially. So, in a sense, I didn't even pursue the PDF solution any further. I wanted to choose PDF over SVG due to the links issue (recall that due to some unresolved bug, see the issue [here](https://bugs.freedesktop.org/show_bug.cgi?id=102882), `pdftocairo` doesn't preserve links in the output; and, yes, I've tried multiple PDF->SVG converters and they either distort the quality/font or remove the links). 

Let's wait and see what others propose with regard to this issue. I'd be very happy to work on another solution.